cmd/sync.go

```diff
@@ -55,4 +58,8 @@ func init() {
 		0, "aritificial delay in seconds that is injected between insert operations \n"+
 			"for related entities (usually for cassandra deployments).\n"+
 			"See 'db_update_propagation' in kong.conf.")
+	syncCmd.Flags().IntVar(&syncCmdRetries, "retries",
+		1, "Number of times decK will retry an insert or update operation")
+	syncCmd.Flags().IntVar(&syncCmdRetryDelay, "retry-delay",
+		0, "Number of seconds decK will wait before retrying an insert or update operation")
```

Comment:
Please do not make retry delay configurable (yet).
Why?
1. Keeping configuration parameters to minimum is important for a good user experience.
2. Configuring delay in such cases seldom adds value. The best practice in this area is to perform an exponential backoff delay which saves the day in a lot of such failure scenarios. Thundering herd is a real problem especially when default parallelism is 10.

Hardcode a delay scheme using a good exponential backoff algorithm library.

Reply:
Makes sense, I changed to an exponential backoff.